### PR TITLE
Fix losant command

### PR DESCRIPTION
**Pull request by heath0xFF**
Fixing `commandReceived` function from: 

```
void commandReceived(String &topic, String &payload) {

  LosantCommand command;
  DynamicJsonDocument root(1024);
  DeserializationError error = deserializeJson(root, payload);

  if (!error) {
    command.name = root["name"];
    command.time = root["$time"];
    JsonObject object = root["payload"].to<JsonObject>();
    command.payload = &object;

    LosantDevice::commandCallback(&command);
  }
}
```

to 

```
void commandReceived(String &topic, String &payload) {

  LosantCommand command;
  DynamicJsonDocument root(1024);
  DeserializationError error = deserializeJson(root, payload);

  if (!error) {
    command.name = root["name"];
    command.time = root["$time"];
    JsonObject object = root["payload"];
    command.payload = &object;

    LosantDevice::commandCallback(&command);
  }
}
```

Change is on `JsonObject object = root["payload"].to<JsonObject>();` to `JsonObject object = root["payload"];`

### CODE REVIEW

**Comment by spacetc62** [comment]
you will need to bump the version in library.json and library.properties

**Comment by heath0xFF** [comment]
> you will need to bump the version in library.json and library.properties

2.0.0 -> 2.0.1, correct?

File: examples/esp8266/esp8266.ino [hunk 1]
```diff
@@ -189,4 +200,4 @@ void loop() {
   }
 
   delay(100);
-}
+}
```
Comment (ajcanterbury) [inline examples/esp8266/esp8266.ino]: Removing the empty EOF breaks many old C/C++ tools, not sure if anyone uses them with an Arduino files but it's probably safer to add it back in.

**Comment by spacetc62** [comment]
> 2.0.0 -> 2.0.1, correct?

yup

File: examples/esp8266/esp8266.ino (hunk 1 above)
Comment (heath0xFF) [reply examples/esp8266/esp8266.ino]: Done!

**Review by spacetc62 — APPROVED** [review]

**Review by ajcanterbury — APPROVED** [review]
Don't have the devices accessible to test this but it looks good.

**Review by anaptfox — APPROVED** [review]
I've tested with the ESP8266

**Pull request merged**